src/lib/Behat/Component/ContentItemAdminPreview.php

```diff
@@ -94,6 +95,16 @@ protected function detectFieldTypeIdentifier(CSSLocator $fieldValueLocator)
         $fieldTypeIdentifierRegex = '/ez|ibexa[a-z_]*-field/';
         preg_match($fieldTypeIdentifierRegex, $fieldClass, $matches);
 
-        return explode('-', $matches[0])[0];
+        if (empty($matches)) {
+            throw new RuntimeException(sprintf(
+                'Cannot match results for pattern: "%s" and subject: "%s".',
+                $fieldTypeIdentifierRegex,
+                $fieldClass
+            ));
+        }
+
+        $matchedResults = explode('-', $matches[0]);
```

Comment:
Are we sure `$matches` isn't empty?

Reply:
I don't understand. This condition checks if matches isn't empty: [R98](https://github.com/ibexa/admin-ui/pull/1336/files/90aa2e579c158103abdffb9170a815b3be476853#diff-8eb03170d2218673c42566233525dbdf178a2e1b7beb06a9b0feafdd129a8983R98), so why might it be empty here?

Reply:
Oh yeah, brainfart most likely. 😅